q2_longitudinal/plugin_setup.py

```diff
@@ -236,6 +240,29 @@
 )
 
 
+plugin.visualizers.register_function(
+    function=anova,
+    inputs={},
+    parameters={'metadata': Metadata,
+                'formula': Str,
+                'sstype': Int % Range(1, 3)},
```

Comment:
My understanding is that this parameter maps to `typ` in the [statsmodels API](https://www.statsmodels.org/stable/generated/statsmodels.stats.anova.anova_lm.html#statsmodels.stats.anova.anova_lm):

> The type of Anova test to perform

What is we made this more clear to users by calling this param `test_type`, and made it `Str % Choices(['fixed', 'random', 'mixed'])`, where those map to `{'I', 'II', 'III'}` respectively?

Ref: https://en.wikipedia.org/wiki/Analysis_of_variance#Classes_of_models

Reply:
> are different. 

I blame their docs:

> The type of Anova test to perform

It should maybe read like:

> Type of sum of squares to use.

---

How about we go with `Str % Choices(['I', 'II', 'III'])`?

Reply:
that sounds good to me (blaming statsmodels and using the roman numeral nomenclature)